package.json

```diff
@@ -1,7 +1,10 @@
 {
   "devDependencies": {
     "@salesforce/dev-config": "1.4.1",
+    "@salesforce/dev-scripts": "0.3.14",
     "ajv": "6.1.1",
+    "commitizen": "^3.0.5",
+    "cz-conventional-changelog": "^3.2.0",
```

Comment:
Note that the default cz-commitizen does not have capabilities for some of the customizations I think we'll want. There is another project, cz-customizable, that I was experimenting with for enforcing scopes and different types. However, this is not approved by security. It sounds like the CLI team will be creating their own adaptor (off-set of cz-commitizen) at which point we might be able to add some of these customizations.

Reply:
Also note, that for adding the work item as a required field, I think we might want to go down the route that the apex-jorje project took. I believe the enforcement of the rev and work item in the final squashed commit comes from a bot named [TOK_sfci12](https://git.soma.salesforce.com/tok-sfci12).